fast_barcode_scanner/lib/src/overlays/code_boundary_overlay/code_border_painter.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/material.dart';
+
+import '../../../fast_barcode_scanner.dart';
+
+class CodeBoarderPainter extends CustomPainter {
```

Comment:
Its 'border' instead of 'boarder', which would be like a Snowboarder.

Reply:
good catch, will fix.